tools/topology/sof-icl-nocodec.m4

```diff
@@ -14,8 +14,8 @@ include(`common/tlv.m4')
 # Include Token library
 include(`sof/tokens.m4')
 
-# Include Icelake DSP configuration
-include(`platform/intel/icl.m4')
+# Include DSP configuration
+include(`platform/intel/'PLATFORM`.m4')
```

Comment:
I assume `platform/intel/jsl.m4` does exist (can't check right now), right?

Reply:
Why not exist? We already have a lot of sof-jsl* tplg

Reply:
>
a better answer:

````bash
ls platform/intel/jsl*
platform/intel/jsl-dedede.m4  platform/intel/jsl.m4  platform/intel/jsl-rt1015.m4
````